python/tensor_sketch_gpu.py

```diff
@@ -0,0 +1,199 @@
+# GPU TENSOR SKETCH
+
+from base import *
+from tensor_sketch import TS
+
+from numba import cuda
+
+
+# CUDA kernel to sketch a list of sequences.
+# A, t, D, L (int32): parameters as usual.
+# global_hashes (int32[:, :]): A*t device array of hashes.
+# global_signs (float32[:, :]): A*t device array of hashes.
```

Comment:
hashes -> signs

Also, why are the signs float32?

Reply:
It's floats because the int32->float32 conversion that's needed later apparently slows down GPU code.
Added that as a comment.